### PR TITLE
antisaccade notebook and initial refactor

### PR DESCRIPTION
**Pull request by scott-huberty**
Hey @jadesjardins I created a notebook specifically for antisaccade (so different team members don't have to change the subject/task when they pull the notebook if they want to work on a different task).

I Also refactored some of the code.. I'll leave comments to those sections for you.

As far as AS - we are close...

The head channels show up and do have the gaps as I'd expect (just visually inspecting). But the eyegaze and pupil channels are missing.

image

### CODE REVIEW

File: q1k_sync_tools.py [hunk 1]
```diff
@@ -2,46 +2,52 @@
...
+               'go', 'plr', 'mmn', 'nsp', 'fsp']
+
+
+def get_event_dict(raw, events):
+
+    stim_names = raw.copy().pick('stim').info['ch_names']
+    event_dict = {event: int(i) + 1
+                  for i, event in enumerate(stim_names)
+                  if event != 'STI 014'}
+
```
Comment (scott-huberty) [inline q1k_sync_tools.py:15]: As I understand it you did

```
for i in range(0,len(stim_names)-1):
    event_dict[stim_names[i]] = i + 1
```
You wanted to create a `dict` with each `stim` channel as a key, and an integer as a value, excluding for the last `stim` channel.. `STI 014`. If True, I'd prefer this method as it is a little more explicit.

File: q1k_sync_tools.py [hunk 2]
```diff
@@ -2,46 +2,52 @@
...
+        event_dict['dstr']
+        print('found dstr label.. removing it')
+        del event_dict['dstr']
+        for k in event_dict:
+            event_dict[k] -= 1
+    except:
+        print('no dstr label found.. continuing')"""
+
+    return event_dict            
+
```
Comment (scott-huberty) [inline q1k_sync_tools.py:42]: Just wrapping that whole commented section in triple quotes instead of having a `#` on each line.

**Comment by scott-huberty** [comment]
Can this be merged?

@jadesjardins, how many changes have you made to your notebooks since you last pushed?

**Comment by scott-huberty** [comment]
Alternatively, If I don't hear back I will assume I have administrator rights and  will merge next week.

**Comment by scott-huberty** [comment]
Hey @jadesjardins let me know if the code in this PR still works for you (I havent' tested in a while). 

On a related note I opened https://github.com/mne-tools/mne-python/pull/12300 with MNE to read EGI events directly as annotations, which should make your lives easier. 

I started a branch for standardizing external pilot files (starting with GO), but havent' gotten around to finishing it yet..

**Comment by jadesjardins** [comment]
yes, thanks... I had been developing things locally in an unorganized way and now I am working on bringing everything together to be current and get q1k BIDS compliant at the onset. The direct read of events to annotations would be great! thanks! I am testing a generic BIDS process to run pylossless.. once that I am happy with that I am going to start working with the events and task specific details. I will take a look at what you have done with GO.

**Comment by scott-huberty** [comment]
Alright I'll try to push that forward for you in the next week or two! If I can't I'll share the WIP work for GO that I have.

**Pull request merged**